"make check" returns pytest Error 127

Hi!

I'm not familiar with pytest and can't seem to make `make check` command in the master branch.

I get instantly : 
```
PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=.:wasp/boards/simulator:wasp:wasp/apps/system \
pytest-3 -v -W ignore  wasp/boards/simulator
/bin/sh: 1: pytest-3: not found
make: *** [Makefile:126: check] Error 127
```

Can anyone give me pointers on this?

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 export PYTHONPATH := $(CURDIR)/tools/nrfutil:$(CURDIR)/tools/intelhex:$(PYTHONPATH)
 
 PYTHON ?= python3
-PYTEST ?= pytest-3
+PYTEST ?= $(PYTHON) -m pytest
 
 all : bootloader reloader micropython
 

